src/components/index.tsx

```diff
@@ -1,17 +1,42 @@
+import { darkTheme, lightTheme, Provider } from "@adobe/react-spectrum";
 import { GoogleOAuthProvider } from "@react-oauth/google";
+import { App } from "components/App";
+import { Luigi } from "components/Luigi";
+import { Stefano } from "components/Stefano";
 import { AuthenticationProvider } from "providers/AuthenticationProvider";
-import { SignatureProvider } from "providers/SignatureProvider";
-import React from "react";
+import { SignatureProvider, useSignatureContext } from "providers/SignatureProvider";
+import React, { FC } from "react";
 import * as ReactDOM from "react-dom";
+import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
 
-import App from "./App/App";
+import { Layout } from "./Layout";
 
-const AppWithProviders = () => {
+const AppRouter: FC = () => {
+    const { isDarkMode } = useSignatureContext();
+
+    return (
+        <Provider theme={isDarkMode ? darkTheme : lightTheme} colorScheme="light" height="100%">
+            <Routes>
+                <Route path="/" element={<Layout />}>
+                    <Route path="/" element={<App />} />
+                    <Route path="luigi" element={<Luigi />} />
+                    <Route path="stefano" element={<Stefano />} />
+                </Route>
+
+                <Route path="*" element={<Navigate to="/" replace />} />
+            </Routes>
+        </Provider>
+    );
+};
+
+const AppWithProviders: FC = () => {
     return (
         <GoogleOAuthProvider clientId="282687558506-tpj65jsu2o26lepumqfkie6drm1p7vni.apps.googleusercontent.com">
             <AuthenticationProvider>
                 <SignatureProvider>
-                    <App />
+                    <BrowserRouter>
```

Comment:
Adding the router to the application

Reply:
vanilla package BrowserRouter wraps the custom AppRouter FC defined here in line 14.